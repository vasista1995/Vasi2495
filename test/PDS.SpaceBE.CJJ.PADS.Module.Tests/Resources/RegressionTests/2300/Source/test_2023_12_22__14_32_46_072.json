{
  "IdSource": "SPACEACT2:CJJ:BE:2Z3490HU:::Force_ShearOnDiode:50248:243031412:0",
  "Data_flat_Metadata": {
    "Machine": "LAW033",
    "Classification": "Routine check(Daily SPC)",
    "Operator": "30156566",
    "Package": "CIPOS MINI",
    "WireSize": "350",
    "ParameterClass": "PPAP",
    "Segment": "IFPS",
    "Module": "MiniFP",
    "Device": "IKCM15F60GA",
    "FourDReport": "*",
    "SpecialCharacteristics": "ISC",
    "WireType": "Al",
    "MaterialType": "Schweizer",
    "Lot": "2Z3490HU",
    "ParameterName": "Force_ShearOnDiode",
    "ParameterUnit": "g",
    "SiteKey": "CJJ",
    "ChannelId": "50248",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-12-10T13:52:59Z",
    "CreatedTimestamp": "2023-12-10T22:52:59Z",
    "UpdatedTimestamp": "2023-12-10T22:52:59Z",
    "UpdatedTimestampUtc": "2023-12-10T13:52:59Z",
    "ExportedTimestampUtc": "2023-12-10T22:52:59Z",
    "ExportedTimestamp": "2023-12-10T22:52:59Z",
    "SampleTimestampUtc": "2023-12-10T13:52:59Z",
    "SampleTimestamp": "2023-12-10T22:52:59Z",
    "SourceDataLevel": "C",
    "ChannelName": "Wedge_ShearForce/MiniFP/Al/350/Diode",
    "RvStoreFlag": "Y",
    "CalcCreated": "09/17/2023 15:17:11",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 601916,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2300",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 550.0,
        "SpecTarget": 1800.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYYNYNNYYYYYYYYYYNNNYYYNYYNNNYNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "1812,66",
        "Median": "1856",
        "Sigma": "191,168546052953",
        "Min": "1545,8",
        "Max": "2043,7",
        "Q1": "1709,7",
        "Q3": "1908,1"
      },
      "ControlLimits": {
        "ctrl_mean_low": 1470.09,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 1938.97,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 2407.85,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "ctrl_sigma_high": 403.93,
        "CtrlSigmaHighEnabled": "Y",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_high": 991.422,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 1868.62,
        "ExtEwmaMV": 1850.95205473874,
        "ExtMsMV": 12.4026529420122,
        "EwmaS": 149.22067809917,
        "EwmaR": 378.545751605987,
        "ExtEwmaMVLCL": 1506.93829148836,
        "ExtEwmaMVCenter": 1730.62,
        "ExtEwmaMVUCL": 1954.30170851164,
        "ExtMaMVLCL": 1561.53252184119,
        "ExtMaMVCenter": 1730.62,
        "ExtMaMVUCL": 1899.70747815881,
        "EwmaSLCL": 82.8956846105419,
        "EwmaSCenter": 170.58678956612,
        "EwmaSUCL": 258.277894521698,
        "EwmaRLCL": 208.162599923911,
        "EwmaRCenter": 413.571264367816,
        "EwmaRUCL": 618.979928811721,
        "ExtMSMVLCL": 0.280194446621961,
        "ExtMSMVCenter": 132.131288680013,
        "ExtMSMVUCL": 530.776503637,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 1000.0,
        "RawUal": 2715.97851193786,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 243031412,
        "Seqnr": 1,
        "Value": 2043.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031412,
        "Seqnr": 2,
        "Value": 1856.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031412,
        "Seqnr": 3,
        "Value": 1908.1,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031412,
        "Seqnr": 4,
        "Value": 1709.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243031412,
        "Seqnr": 5,
        "Value": 1545.8,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CJJ:BE:::Force_ShearOnDiode:50248",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:2Z3490HU",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CJJ:BE:2Z3490HU:::Force_ShearOnDiode:50248:243031412:0"
  }
}