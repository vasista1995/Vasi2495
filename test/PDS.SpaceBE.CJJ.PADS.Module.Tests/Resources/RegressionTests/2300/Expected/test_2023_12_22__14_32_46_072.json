{
  "Data_flat_Metadata": {
    "Classification": "Routine check(Daily SPC)",
    "MeasLot": "2Z3490HU",
    "SiteKey": "CJJ",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-12-10T13:52:59Z",
    "BeginTimestamp": "2023-12-10T22:52:59Z",
    "EndTimestampUtc": "2023-12-10T13:52:59Z",
    "EndTimestamp": "2023-12-10T22:52:59Z"
  },
  "Data_flat_StrangeMetadata": {
    "WireSize": "350",
    "WireType": "Al",
    "Operator": "30156566",
    "Package": "CIPOS MINI",
    "Device": "IKCM15F60GA",
    "MaterialType": "Schweizer"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 1,
          "SampleId": 243031412,
          "Value": 2043.7,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-10T22:52:59Z",
          "SampleTimestampUtc": "2023-12-10T13:52:59Z",
          "CreatedTimestamp": "2023-12-10T22:52:59Z",
          "CreatedTimestampUtc": "2023-12-10T13:52:59Z",
          "UpdatedTimestamp": "2023-12-10T22:52:59Z",
          "UpdatedTimestampUtc": "2023-12-10T13:52:59Z",
          "NumViolations": 0,
          "SampleMean": 1812.66,
          "SampleMin": 1545.8,
          "SampleMax": 2043.7,
          "SampleMedian": 1856.0,
          "SampleStdev": 191.168546052953,
          "SampleQ1": 1709.7,
          "SampleQ3": 1908.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 2,
          "SampleId": 243031412,
          "Value": 1856.0,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-10T22:52:59Z",
          "SampleTimestampUtc": "2023-12-10T13:52:59Z",
          "CreatedTimestamp": "2023-12-10T22:52:59Z",
          "CreatedTimestampUtc": "2023-12-10T13:52:59Z",
          "UpdatedTimestamp": "2023-12-10T22:52:59Z",
          "UpdatedTimestampUtc": "2023-12-10T13:52:59Z",
          "NumViolations": 0,
          "SampleMean": 1812.66,
          "SampleMin": 1545.8,
          "SampleMax": 2043.7,
          "SampleMedian": 1856.0,
          "SampleStdev": 191.168546052953,
          "SampleQ1": 1709.7,
          "SampleQ3": 1908.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 3,
          "SampleId": 243031412,
          "Value": 1908.1,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-10T22:52:59Z",
          "SampleTimestampUtc": "2023-12-10T13:52:59Z",
          "CreatedTimestamp": "2023-12-10T22:52:59Z",
          "CreatedTimestampUtc": "2023-12-10T13:52:59Z",
          "UpdatedTimestamp": "2023-12-10T22:52:59Z",
          "UpdatedTimestampUtc": "2023-12-10T13:52:59Z",
          "NumViolations": 0,
          "SampleMean": 1812.66,
          "SampleMin": 1545.8,
          "SampleMax": 2043.7,
          "SampleMedian": 1856.0,
          "SampleStdev": 191.168546052953,
          "SampleQ1": 1709.7,
          "SampleQ3": 1908.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 4,
          "SampleId": 243031412,
          "Value": 1709.7,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-10T22:52:59Z",
          "SampleTimestampUtc": "2023-12-10T13:52:59Z",
          "CreatedTimestamp": "2023-12-10T22:52:59Z",
          "CreatedTimestampUtc": "2023-12-10T13:52:59Z",
          "UpdatedTimestamp": "2023-12-10T22:52:59Z",
          "UpdatedTimestampUtc": "2023-12-10T13:52:59Z",
          "NumViolations": 0,
          "SampleMean": 1812.66,
          "SampleMin": 1545.8,
          "SampleMax": 2043.7,
          "SampleMedian": 1856.0,
          "SampleStdev": 191.168546052953,
          "SampleQ1": 1709.7,
          "SampleQ3": 1908.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 5,
          "SampleId": 243031412,
          "Value": 1545.8,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-10T22:52:59Z",
          "SampleTimestampUtc": "2023-12-10T13:52:59Z",
          "CreatedTimestamp": "2023-12-10T22:52:59Z",
          "CreatedTimestampUtc": "2023-12-10T13:52:59Z",
          "UpdatedTimestamp": "2023-12-10T22:52:59Z",
          "UpdatedTimestampUtc": "2023-12-10T13:52:59Z",
          "NumViolations": 0,
          "SampleMean": 1812.66,
          "SampleMin": 1545.8,
          "SampleMax": 2043.7,
          "SampleMedian": 1856.0,
          "SampleStdev": 191.168546052953,
          "SampleQ1": 1709.7,
          "SampleQ3": 1908.1,
          "SampleSize": 5
        }
      ],
      "process_equipment": "LAW033",
      "ParameterClass": "PPAP",
      "Segment": "IFPS",
      "Module": "MiniFP",
      "FourDReport": "*",
      "SpecialCharacteristics": "ISC",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "N",
          "SpecLowEnabled": "Y",
          "spec_low": 550.0,
          "target": 1800.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 1470.09,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 1938.97,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 2407.85,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "ctrl_stdev_high": 403.93,
          "CtrlStdevHighEnabled": "Y",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_high": 991.422,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 1812.6599999999999,
        "median": 1856.0,
        "stdev": 170.9863456536808,
        "range": 497.9000000000001,
        "min": 1545.8,
        "max": 2043.7,
        "q02": 1558.912,
        "q05": 1578.5800000000002,
        "q25": 1709.7,
        "q75": 1908.1,
        "q95": 2016.58,
        "q98": 2032.852,
        "NumViolations": 0,
        "Samples": "243031412"
      },
      "ParameterName": "Force_ShearOnDiode",
      "ParameterUnit": "g",
      "ChannelId": "50248",
      "SourceDataLevel": "C",
      "channel_name": "Wedge_ShearForce MiniFP Al 350 Diode",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-12-10T13:52:59Z",
      "created_timestamp": "2023-12-10T22:52:59Z",
      "updated_timestamp_utc": "2023-12-10T13:52:59Z",
      "updated_timestamp": "2023-12-10T22:52:59Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-12-10T13:52:59Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:CJJ:BE:::",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:2Z3490HU",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "CJJ:BE:2Z3490HU:::Force_ShearOnDiode:50248:243031412:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-12-10T22:52:59Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot:2Z3490HU:SPACEAGGED2:CJJ:BE::::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "CJJ"
  }
}